webapp/src/ts/services/tasks-for-contact.service.ts

```diff
@@ -15,9 +15,9 @@ export class TasksForContactService {
 
   private getIdsForTasks(model) {
     const contactIds = [];
-    if (!model.type.person && model.children) {
+    if (!model?.type?.person && model?.children) {
       model.children.forEach(child => {
-        if (child.type.person && child.contacts && child.contacts.length) {
+        if (child?.type?.person && child?.contacts && child?.contacts?.length) {
```

Comment:
I think `if (child?.type?.person && child?.contacts?.length)` is sufficient, is it not?

Reply:
It is. I have made the update.